src/plugins/collision/collider/mod.rs

```diff
@@ -92,209 +91,457 @@ pub trait ScalableCollider: AnyCollider {
+    /// Constructs a collider with [`Collider::regular_polygon`].
+    #[cfg(feature = "2d")]
+    RegularPolygon { circumradius: f32, sides: usize },
+    /// Constructs a collider with [`Collider::polyline`].
+    Polyline {
+        vertices: Vec<Vector>,
+        indices: Option<Vec<[u32; 2]>>,
+    },
+    /// Constructs a collider with [`Collider::trimesh`].
+    Trimesh {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 3]>,
+    },
+    /// Constructs a collider with [`Collider::trimesh_with_config`].
+    TrimeshWithConfig {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 3]>,
+        flags: TriMeshFlags,
+    },
+    /// Constructs a collider with [`Collider::convex_decomposition`].
+    #[cfg(feature = "2d")]
+    ConvexDecomposition {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 2]>,
+    },
+    /// Constructs a collider with [`Collider::convex_decomposition`].
+    #[cfg(feature = "3d")]
+    ConvexDecomposition {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 3]>,
+    },
+    /// Constructs a collider with [`Collider::convex_decomposition_with_config`].
+    #[cfg(feature = "2d")]
+    ConvexDecompositionWithConfig {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 2]>,
+        params: VhacdParameters,
+    },
+    /// Constructs a collider with [`Collider::convex_decomposition_with_config`].
+    #[cfg(feature = "3d")]
+    ConvexDecompositionWithConfig {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 3]>,
+        params: VhacdParameters,
+    },
+    /// Constructs a collider with [`Collider::convex_hull`].
+    #[cfg(feature = "2d")]
+    ConvexHull { points: Vec<Vector> },
+    /// Constructs a collider with [`Collider::convex_hull`].
+    #[cfg(feature = "3d")]
+    ConvexHull { points: Vec<Vector> },
+    /// Constructs a collider with [`Collider::heightfield`].
+    #[cfg(feature = "2d")]
+    Heightfield { heights: Vec<Scalar>, scale: Vector },
+    /// Constructs a collider with [`Collider::heightfield`].
+    #[cfg(feature = "3d")]
+    Heightfield {
+        heights: Vec<Vec<Scalar>>,
+        scale: Vector,
+    },
+    /// Constructs a collider with [`Collider::trimesh_from_mesh`].
+    #[cfg(all(feature = "3d", feature = "collider-from-mesh"))]
     #[default]
-    TriMesh,
-    /// A triangle mesh with a custom configuration.
+    TrimeshFromMesh,
+    /// Constructs a collider with [`Collider::trimesh_from_mesh_with_config`].
     #[cfg(all(
-        feature = "default-collider",
-        any(feature = "parry-f32", feature = "parry-f64")
+        feature = "3d",
+        feature = "collider-from-mesh",
+        feature = "default-collider"
     ))]
-    TriMeshWithFlags(TriMeshFlags),
-    /// A convex hull.
-    ConvexHull,
-    /// A compound shape obtained from a decomposition into convex parts using the specified
-    /// [`VHACDParameters`].
+    TrimeshFromMeshWithConfig(TriMeshFlags),
```

Comment:
And these should be `TriMesh` too

Reply:
Changed all to `Trimesh` instead per discussion on Discord [`526135e` (#378)](https://github.com/Jondolf/bevy_xpbd/pull/378/commits/526135e1fa5d37b1a1616b7a383574b5768f8284)